src/rpc/core_rpc_server_commands_defs.h

```diff
@@ -176,6 +176,344 @@ namespace cryptonote
   };
 
   //-----------------------------------------------
+  struct COMMAND_RPC_LIGHT_WALLET_GET_ADDRESS_TXS
+  {
+      struct request
+      {
+        std::string address;
+        std::string view_key;
+
+        BEGIN_KV_SERIALIZE_MAP()
+          KV_SERIALIZE(address)
+          KV_SERIALIZE(view_key)
+        END_KV_SERIALIZE_MAP()
+      };
+      
+      struct spent_output {
+        std::string amount;
+        std::string key_image;
+        std::string tx_pub_key;
+        int out_index;
+        int mixin;
+        
+        
+        BEGIN_KV_SERIALIZE_MAP()
+          KV_SERIALIZE(amount)
+          KV_SERIALIZE(key_image)
+          KV_SERIALIZE(tx_pub_key)
+          KV_SERIALIZE(out_index)
+          KV_SERIALIZE(mixin)
+        END_KV_SERIALIZE_MAP()
+      };
+      
+      struct transaction
+      {
+        int id;
```

Comment:
@moneromooo-monero commented on this once, but `int` range varies by platform. Please use "specific" (`uint16_t`, etc.) unsigned types for all of the ints in this file.